package-lock.json

```diff
@@ -0,0 +1,4625 @@
+{
```

Comment:
Could you add `.npmrc` with `package-lock=false` and remove this `package-lock.json`?
See also: https://github.com/vuejs/eslint-plugin-vue/pull/128

Reply:
Package lock is included to prevent master from failing without any changes

Reply:
I want not to merge `package-lock.json` into this repo because it damages the value of CI builds, the file disallows CI builds to run tests with dependencies that users install actually.